norse/torch/module/lif.py

```diff
@@ -53,14 +57,23 @@ class LIFCell(SNNCell):
         dt (float): Time step to use. Defaults to 0.001.
     """
 
-    def __init__(self, p: LIFParameters = LIFParameters(), sparse=False, **kwargs):
+    def __init__(
+        self, p: LIFParameters = LIFParameters(), sparse=False, adjoint=False, **kwargs
+    ):
         if sparse:
```

Comment:
I would suggest we merge this with #230, but perhaps into something similar to:
```python
activation = ...
super().__init__(activation=activation, ...)
```
That'll reduce a lot of redundant if-else blocks

Reply:
Another idea could be to wrap the modules instead of adding flags. The coexisting of the two flags indicate that it's possible to have both sparse *and* adjoint. We could perhaps do something like this:
```
class LIFCell(SNNCell):
    def__init__(self, activation=lif_feed_forward_step, ...):
        ...
class LIFCellAdjoint(LIFCell):
    def __init__(self):
        super().__init__(activation=lif_feed_forward_adjoint_step, ...)
class LIFCellSparse(LIFCell):
    def __init__(self):
         super().__init__(activation=lif_feed_forward_adjoint_step, ...)
```         
Alternatively, we can add an Enum or similar in the construction to ensure that the two flags aren't activated simultaneously.

Basically, I think we should think a bit about how we "toggle" the different activation modes of a module.